speech/samples/v1p1beta1/samplegen_read_and_write_files.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_read_and_write_files")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-speech
+
+# sample-metadata
+#   title: Read binary file into bytes field & write string in response to file
+#   description: Read binary file into bytes field & write string in response to file
+#   usage: python3 samples/v1p1beta1/samplegen_read_and_write_files.py
+
+# [START samplegen_read_and_write_files]
+from google.cloud import speech_v1p1beta1
+from google.cloud.speech_v1p1beta1 import enums
+import io
+
+
+def sample_recognize():
+    """Read binary file into bytes field & write string in response to file"""
+
+    client = speech_v1p1beta1.SpeechClient()
+
+    encoding = enums.RecognitionConfig.AudioEncoding.MP3
+    config = {"encoding": encoding}
+
+    # The bytes from this file will be read into `content`
+    with io.open("path/to/file.mp3", "rb") as f:
+        content = f.read()
+    audio = {"content": content}
+
+    response = client.recognize(config, audio)
+    # Your audio has been transcribed.
+    # Writing audio transcript to transcript.txt for demonstration:
+    with open("transcript.txt", "w") as out:
+        out.write(response.results[0].alternatives[0].transcript)
+
+
+# [END samplegen_read_and_write_files]
+
+
+def main():
```

Comment:
Maybe skip having a separate function and put this down below where line 60 is now

Reply:
Good thought, having the method -vs- not having the method is something I've seen both of.

I direct you to this comment (on the same or) => [Method or No Method](#discussion_r357394695) (_that is the question..._)

Reply:
Hiding this thread in favor of [this one above (duplicate)](https://github.com/googleapis/google-cloud-python/pull/9970#discussion_r357394695)